tic-tac-toe-the3rd/src/components/Game.js

```diff
@@ -0,0 +1,96 @@
+import React, { useContext } from 'react';
+import Cells from './Cell';
+import BoardContext from '../context/BoardContext';
+
+
+function Board(props) {
+  const { board, setBoard, turn, setTurn, history, setHistory } = useContext(BoardContext);
+
+  function declareWinner(board) {
+    const winningConditions = [
+      [0, 1, 2],
+      [3, 4, 5],
+      [6, 7, 8],
+      [0, 3, 6],
+      [1, 4, 7],
+      [2, 5, 8],
+      [0, 4, 8],
+      [2, 4, 6]
+    ];
+
+    for (let i = 0; i < winningConditions.length; i++) {
+      const [a, b, c] = winningConditions[i];
+      if (board[a] && board[a] === board[b] && board[a] === board[c]) {
+        return board[a];
+      }
+    }
+    return null;
+
+  }
+  const winner = declareWinner(board);
+  let status;
+
+  if (winner) {
+    status = `The winner is ${winner}`;
+  }
+
+  function clickHandler(num) {
```

Comment:
I'd probably call this function `move` or `makeTurn` instead of `clickHandler` since that's a little bit more descriptive.